rust/main/hyperlane-base/src/types/gcs_storage.rs

```diff
@@ -241,8 +241,9 @@ impl CheckpointSyncer for GcsStorageClient {
         signed_checkpoint: &SignedCheckpointWithMessageId,
     ) -> Result<()> {
         let object_name = Self::get_checkpoint_key(signed_checkpoint.value.index);
+        let full_path = self.object_path(&object_name);
```

Comment:
maybe rename `object_name` (on line 242) to `object_key` and `full_path` to `object_name` to avoid confusion, since the methods below use `object_name` with a different purpose than here